### PR TITLE
Responsive Header and Search Area

### LINKED ISSUE #19: Design for mobile
In #15 we got the basic site, but didn't do any explicit work with a responsive design for mobile viewports. In particular, the header component is broken on mobile. Let's fix that up.

### PR DESCRIPTION
**Pull request by BhavyaT-135**
Fixes #19 

In the above pull request, I updated the header section to fit into the responsive design. Thereafter, the search box and search area results were aligned and modified to match the header section.

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -56,6 +68,16 @@ const NavItem = styled.div`
...
   }
+
+  @media screen and (max-width: 600px) {
+    width: 100vw;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-top: 10px;
+    padding-left: 3rem;
+    ${'' /* Or .5 rem for the left padding */}
```
Comment (jonathansick) [inline src/components/header.js:79]: Yeah I think I like `padding-left: 0.5 rem` to line up with the logo rather than lining up on "Learn.Astropy."

**Review by jonathansick — COMMENTED** [review]
Sorry @BhavyaT-135, we've been away from the project this summer so I haven't got to your PR sooner. I like this, and it's definitely makes the site immediately usable on phones. Down the line I think I'll want to give phone users a way to use the search refinements; but this is good for now.

The only change I'd suggest is the left padding in the Header. Would you like to change that? Then I'll merge this. Thanks again!

**Comment by BhavyaT-135** [comment]
@jonathansick Thank you for the quick response. I've added the left padding in the header section as required. Kindly check. 😊

**Comment by jonathansick** [comment]
Thank you @BhavyaT-135 ! 🎉

**Pull request merged**